bots/tests-invoke

```diff
@@ -79,7 +80,13 @@ class PullTask(object):
         return "{0} {1} {2}".format(self.name, self.context, self.revision)
 
     def start_publishing(self, host, api):
-        identifier = self.name + "-" + self.revision[0:8] + "-" + self.context.replace("/", "-")
+        identifier = "-".join([
+            self.name,
+            self.revision[0:8],
+            self.context.replace("/", "-"),
+            str(int(time.time()))
```

Comment:
While we are at making stuff friendlier for humans, how about `time.strftime('%Y%m%d-%H%M%M')` instead, so that it's simple to see to which date/time they apply to?